payments-core/src/main/java/com/stripe/android/payments/paymentlauncher/PaymentLauncher.kt

```diff
@@ -87,25 +87,18 @@ interface PaymentLauncher {
          * recompositions.
          */
         @Deprecated(
-            "This method creates a new PaymentLauncher object every time it is called, even" +
-                "during recompositions.",
-            replaceWith = ReplaceWith("PaymentLauncher.rememberLauncher(publishableKey, stripeAccountId, callback)")
+            message = "Use rememberPaymentLauncher() instead",
+            replaceWith = ReplaceWith(
+                expression = "rememberPaymentLauncher(publishableKey, stripeAccountId, callback)",
+            ),
         )
         @Composable
         fun createForCompose(
             publishableKey: String,
             stripeAccountId: String? = null,
             callback: PaymentResultCallback
         ): PaymentLauncher {
-            val context = LocalContext.current
-            return PaymentLauncherFactory(
-                context = context,
-                hostActivityLauncher = rememberLauncherForActivityResult(
-                    PaymentLauncherContract(),
-                    callback::onPaymentResult
-                ),
-                statusBarColor = context.findActivity()?.window?.statusBarColor,
-            ).create(publishableKey, stripeAccountId)
+            return rememberPaymentLauncher(publishableKey, stripeAccountId, callback)
```

Comment:
Just because it’s deprecated doesn’t mean that we can’t fix the faulty implementation 🤷‍♂️